### PR TITLE
LPS-109804 rename

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related issue: https://issues.liferay.com/browse/LPS-109804

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_8_0/UpgradeDDMStructure.java [hunk 1]
```diff
@@ -182,53 +209,81 @@ private void _upgradeDDMStructureLayout() throws Exception {
...
-						_upgradeDDMStructureLayoutDefinition(
-							definition, structureVersionId));
+					Exception exception =
+						ddmFormLayoutDeserializerDeserializeResponse.
+							getException();
 
-					long structureLayoutId = rs.getLong("structureLayoutId");
+					if (exception != null) {
+						throw new UpgradeException(exception);
+					}
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_8_0/UpgradeDDMStructure.java:254]: It sounds like a good idea @natocesarrego.

[the remark this answers is not in the document]
Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_8_0/UpgradeDDMStructure.java:254]: It sounds good to me too, @natocesarrego. I'll create that in next pull :D

**Pull request closed without merging**